easybuild/easyconfigs/r/RNAclust/RNAclust-1.3-foss-2016b-Perl-5.24.0.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'RNAclust'
+version = '1.3'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.bioinf.uni-leipzig.de/~kristin/Software/%(name)s/'
+description = """RNAclust is a perl script summarizing all the single steps required for clustering
+ of structured RNA motifs, i.e. identifying groups of RNA sequences sharing a secondary structure
+ motif. It requires as input a multiple FASTA file."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.bioinf.uni-leipzig.de/~kristin/Software/%(name)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['c86becf9aafd2d78350bc50ea597b5c6c6bf1b0e5fc74919d0a5d9ba91c22817']
+
+dependencies = [
+    ('Perl', '5.24.0'),
+    ('LocARNA', '1.9.2'),
+    ('ViennaRNA', '2.3.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s.pl'],
```

Comment:
@SethosII Since stuff is getting compiled, I would check (some of) the binaries here too?

`bin/pgma`, `bin/rnaclustDecodeTable`, `bin/RNAsoup` are the binaries I see

Reply:
@boegel I changed this.